_posts/2020-8-12-new-release-2020-3.md

```diff
@@ -0,0 +1,22 @@
+---
+title:  "Connector SDK now targets Tableau 2020.3"
+abstract: "The master SDK branch now targets Tableau 2020.3. The Connector SDK for 2020.2 has released for those wishing to target that version."
+---
+
+On August 12, with the full release of Tableau 2020.3 we moved the latest version of the master branch to target Tableau 2020.3. This means that the samples and connectors made with this version of the SDK may not work with older versions of Tableau. To develop against Tableau 2020.2, please download the [Tableau Connector SDK for Tableau 2020.2](https://github.com/tableau/connector-plugin-sdk/releases/tag/tableau-2020.2) release.
+
+Features included now included in the SDK for Tableau 2020.3 include:
+
+- __Connection Dialogs v2__ <br/>
+Connection Dialogs v2 feature, which will allow for more flexible connection dialogs. Learn more about connection Dialogs v2 [here]({{ site.baseurl }}/docs/mcd).
+- __Updated Dialect Samples__ <br/>
+The 2020-3 branch of connector SDK contains the complete dialect definition file for [Postgres JDBC connector](https://github.com/tableau/connector-plugin-sdk/blob/dev-2020.3/samples/plugins/postgres_jdbc/dialect.tdd) and [Postgres ODBC connector](https://github.com/tableau/connector-plugin-sdk/blob/dev-2020.3/samples/plugins/postgres_odbc/dialect.tdd). This should serve as a good example to write a dialect file for JDBC or ODBC connectors with 2020.3 updates.
+- __Delegation UID__ <br/>
+This applies to databases that support a user(authenticated_user) that can delegate requests to another user(delegated_user) by enabling the client to pass a DelegationUID(delegated_user) to the database server. More details on this feature will be added with the release of 2020.3.
```

Comment:
Since this is 2020.3 might remove "More details on this feature will be added with the release of 2020.3."

Reply:
Ack, copy/paste error. Thanks for catching.